CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(PACKAGE_BUGREPORT "https://github.com/googleapis/google-cloud-cpp/issues")
 
 project(
     google-cloud-cpp
-    VERSION 2.4.0
+    VERSION 2.5.0
```

Comment:
Did we want to try `2.5.0-rc` ?

Reply:
Also do you want to add the new version to the CHANGELOG as a placeholder in this PR?

Reply:
> Did we want to try `2.5.0-rc` ?

I'll be working on that separately.  Yes, that means we won't be able to trust "-rc" until 2.6.

Reply:
> Also do you want to add the new version to the CHANGELOG as a placeholder in this PR?

Oops.  Hoist by my own petard.  Fixed.